wrappers/awskms/awskms.go

```diff
@@ -10,20 +10,20 @@ import (
 	"os"
 	"sync/atomic"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/kms"
-	"github.com/aws/aws-sdk-go/service/kms/kmsiface"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
 	cleanhttp "github.com/hashicorp/go-cleanhttp"
 	"github.com/hashicorp/go-hclog"
 	wrapping "github.com/hashicorp/go-kms-wrapping/v2"
-	"github.com/hashicorp/go-secure-stdlib/awsutil"
+	"github.com/hashicorp/go-secure-stdlib/awsutil/v2"
 )
 
 // These constants contain the accepted env vars; the Vault one is for backwards compat
 const (
 	EnvAwsKmsWrapperKeyId   = "AWSKMS_WRAPPER_KEY_ID"
 	EnvVaultAwsKmsSealKeyId = "VAULT_AWSKMS_SEAL_KEY_ID"
+	EnvAwsKmsEndpoint       = "AWS_KMS_ENDPOINT"
```

Comment:
I see below that we need to keep this environment variable, but could we perhaps also look for `AWSKMS_ENDPOINT` and deprecate `AWS_KMS_ENDPOINT`? It's unfortunate that the formatting differs to `AWSKMS_WRAPPER_KEY_ID` IMO.